wss/1.0.0/modules/ROOT/pages/index.adoc

```diff
@@ -24,15 +23,27 @@ Before creating an app, you must have access to:
 
 * The WSS Module target resource
 * Anypoint Platform
-* Anypoint Studio version <x.x> or later
-*// If your connector has other prerequisites specific to this connector, list them here.
+* Anypoint Studio version 7.x or later
+* Web Services Security (WSS) extension for SOAP.
 
 == Common Use Cases For the Connector
 
-// List common use cases for the WSS Module and provide
-// a link to the xref:wss-examples.adoc[Examples] use cases.
+The WSS module enables you to validate WSS headers in the inbound SOAP requests to your application to:
 
-These are some common use cases for WSS Module...
+* Decrypt a SOAP envelope according to the encryption information.
+* Verify a signature validity.
+* Verify the freshness of the message by `timestamp` information.
+* Verify `Username` and `Password` credentials of a `UsernameToken`.
+* Verify `LDAP` credentials of a `UsernameToken`.
```

Comment:
```suggestion
* Verify the `LDAP` credentials of `UsernameToken`
```

Reply:
It's a subtle thing, but for me, you "verify" something to be as expected, as appropriate, or as true.
I would not use "verify" to discover something negative.
Hmm... so I looked it up:

As verbs the difference between verify and determine
is that verify is to substantiate or prove the **truth** of something while determine is to set the **limits** of.